Mage/src/main/java/mage/target/TargetImpl.java

```diff
@@ -203,7 +203,9 @@ public boolean isRequired() {
 
     @Override
     public boolean isRequired(Ability ability) {
-        return ability == null || ability.isActivated() || !(ability.getAbilityType() == AbilityType.SPELL || ability.getAbilityType() == AbilityType.ACTIVATED);
+        return ability == null
+                || ability.isActivated()
+                || !(ability.getAbilityType() == AbilityType.SPELL || ability.getAbilityType().isActivatedAbility());
```

Comment:
changed logic here, before mana abilities required, but now not? is it correct now?

Reply:
By CR definition, mana abilities do not target so the extra check excluding MANA is not needed.